content/docs/reference-test-renderer.md

```diff
@@ -94,157 +97,159 @@ expect(testInstance.findByProps({className: "sub"}).children).toEqual(['Sub']);
 * [`testInstance.parent`](#testinstanceparent)
 * [`testInstance.children`](#testinstancechildren)
 
-## Reference {#reference}
+## Başvuru Dokümanı {#reference}
 
 ### `TestRenderer.create()` {#testrenderercreate}
 
 ```javascript
 TestRenderer.create(element, options);
 ```
 
-Create a `TestRenderer` instance with the passed React element. It doesn't use the real DOM, but it still fully renders the component tree into memory so you can make assertions about it. The returned instance has the following methods and properties.
+Geçirilen React elemaniyla bir `Test Render Edici` nesnesi oluşturun. Bu `Test Render Edici` nesnesi gerçek DOM'u kullanmaz, ancak bileşen ağacını yinede eksiksiz olarak belleğe render eder ve bu konuda doğrulama yapabilirsiniz. Oluşturulan bu nesne, aşağıdaki methodlara ve özelliklere sahiptir.
 
 ### `testRenderer.toJSON()` {#testrenderertojson}
 
 ```javascript
 testRenderer.toJSON()
 ```
 
-Return an object representing the rendered tree. This tree only contains the platform-specific nodes like `<div>` or `<View>` and their props, but doesn't contain any user-written components. This is handy for [snapshot testing](https://facebook.github.io/jest/docs/en/snapshot-testing.html#snapshot-testing-with-jest).
+Render edilen ağacı temsil eden bir object return edin. Bu ağaç yalnızca `<div>` veya `<View>` gibi platforma özgü düğümleri ve bunların props larını içerir, ancak kullanıcı tarafından yazılmış bileşenleri içermez. Bu, [anlık görüntü](https://facebook.github.io/jest/docs/en/snapshot-testing.html#snapshot-testing-with-jest) testi için kullanışlıdır.
```

Comment:
Suggestion:
> İşlenen ağacı temsil eden bir nesne döndürün. Bu ağaç yalnızca `<div>` veya `<View>` gibi platforma özgü birimleri ve bunların bileşenlerini içerir, ancak kullanıcı tarafından yazılmış bileşenleri içermez.Bu, [anlık görüntü (snapshot)](https://facebook.github.io/jest/docs/en/snapshot-testing.html#snapshot-testing-with-jest) testi için kullanışlıdır.

We can leave snapshot as it is for now. When we decide an accepted translation for it, we can update here.